benchmarks/README.md

```diff
@@ -0,0 +1,81 @@
+# FastTrack ML Benchmark Suite
+
+## Introduction
+
+FastTrack ML Benchmark Suite is a project designed to provide a comprehensive and user-friendly performance tracking system for FastTrack ML, with a specific focus on comparing its capabilities with other popular machine learning parameter servers, such as MLflow. This documentation aims to guide users on how to use FastTrack ML Performance Tracker effectively.
```

Comment:
I'd try to avoid the terms "tracking" and "tracker" for the benchmark suite -- FastTrack is the tracker, this is performance benchmarking.